[2024-12-07T07:24:26.843+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-07T07:24:26.903+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: parallel_financial_reports.financial_reports.run_main_report manual__2024-12-07T07:24:17.342746+00:00 [queued]>
[2024-12-07T07:24:26.924+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: parallel_financial_reports.financial_reports.run_main_report manual__2024-12-07T07:24:17.342746+00:00 [queued]>
[2024-12-07T07:24:26.924+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-07T07:24:27.012+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): financial_reports.run_main_report> on 2024-12-07 07:24:17.342746+00:00
[2024-12-07T07:24:27.044+0000] {logging_mixin.py:190} WARNING - /usr/local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=223) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-07T07:24:27.048+0000] {standard_task_runner.py:72} INFO - Started process 227 to run task
[2024-12-07T07:24:27.047+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_financial_reports', 'financial_reports.run_main_report', 'manual__2024-12-07T07:24:17.342746+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/Extract_history_data.py', '--cfg-path', '/tmp/tmp360y_f63']
[2024-12-07T07:24:27.057+0000] {standard_task_runner.py:105} INFO - Job 37: Subtask financial_reports.run_main_report
[2024-12-07T07:24:27.466+0000] {task_command.py:467} INFO - Running <TaskInstance: parallel_financial_reports.financial_reports.run_main_report manual__2024-12-07T07:24:17.342746+00:00 [running]> on host e645099c8412
[2024-12-07T07:24:27.877+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflowDWH' AIRFLOW_CTX_DAG_ID='parallel_financial_reports' AIRFLOW_CTX_TASK_ID='financial_reports.run_main_report' AIRFLOW_CTX_EXECUTION_DATE='2024-12-07T07:24:17.342746+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-07T07:24:17.342746+00:00'
[2024-12-07T07:24:27.883+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-07T07:25:27.434+0000] {logging_mixin.py:190} INFO - Dữ liệu Cân Đối Kế Toán cho mã AGG thiếu từ các năm: 2017.
[2024-12-07T07:25:51.549+0000] {logging_mixin.py:190} INFO - Dữ liệu Kết Quả Kinh Doanh cho mã AGG thiếu từ các năm: 2017.
[2024-12-07T07:25:59.989+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2024-12-07T07:26:00.066+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-12-07T07:26:00.083+0000] {process_utils.py:132} INFO - Sending 15 to group 227. PIDs of all processes in the group: [227]
[2024-12-07T07:26:00.086+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 227
[2024-12-07T07:26:00.092+0000] {taskinstance.py:3093} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-12-07T07:26:00.547+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-07T07:26:00.624+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=227, status='terminated', exitcode=0, started='07:24:26') (227) terminated with exit code 0
